typeql/language/update.feature

```diff
@@ -50,6 +50,33 @@ Feature: TypeQL Update Query
     Given session opens transaction of type: write
 
 
+  Scenario: Deleting anonymous variables throws an exception
+    Given get answers of typeql insert
+      """
+      insert
+        $x isa person, has name "Alex", has ref 0;
+        $y isa person, has name "Alex", has ref 1;
+        (friend: $y) isa friendship, has ref 2;
+      """
+    Given transaction commits
+    Given session opens transaction of type: write
+    Then typeql update; throws exception
+      """
+      match
+      $x isa person, has ref 1;
+      delete $x has name "Alex";
+      insert $x has name "Bob";
+      """
+    Then typeql update; throws exception
```

Comment:
Couldn't this exception occur simply because the transaction has been closed?

I think we need an additional `Given session opens transaction of type: write`